west.yml

```diff
@@ -251,6 +251,16 @@ manifest:
       revision: 384a091445c57b44ac8cbd18ebd245b47c71db94
       import:
         path-prefix: tools
+    - name: bme68x
+      url: https://github.com/boschsensortec/Bosch-BME68x-Library
+      path: modules/lib/bme68x
+      revision: v1.1.40407
+    - name: bsec
+      url: https://github.com/boschsensortec/Bosch-BSEC2-Library
```

Comment:
Please use the previously created remote

Reply:
done